tests/ci/azureml_tests/submit_groupwise_azureml_pytest.py

```diff
@@ -323,19 +338,19 @@ def create_arg_parser():
     parser.add_argument(
         "--subid", action="store", default="123456", help="Azure Subscription ID"
     )
-    # ./reco.yaml is created in the azure devops pipeline.
+    # reco wheel is created in the GitHub action workflow.
     # Not recommended to change this.
     parser.add_argument(
-        "--condafile",
+        "--wheelfile",
         action="store",
-        default="./reco.yaml",
-        help="file with environment variables",
+        default="./dist/recommenders-1.0.0-py3-none-any.whl",
+        help="recommenders whl file path",
```

Comment:
@miguelgfierro on GitHub machines, I build a wheel from the checked out repo and `wheelfile` used to be the path to the wheel file (used to set up AzureML environment). The wheel file name generated by bdist command is not constant, so this is how I locate the wheel file now: https://github.com/microsoft/recommenders/blob/53c01881843b256ab33bffffa91bdf6e363e6f6a/tests/ci/azureml_tests/submit_groupwise_azureml_pytest.py#L464
I will remove the `wheelfile` argument in another PR since this argument is not being used anymore.